packages/backend/src/errors/generate-error-email.ts

```diff
@@ -0,0 +1,53 @@
+import { DateTime } from 'luxon'
+
+import { sendEmail } from '@/helpers/send-email'
+
+const MAX_LENGTH = 80
+
+function truncateFlowName(flowName: string) {
+  return flowName.length > MAX_LENGTH
+    ? flowName.slice(0, MAX_LENGTH) + '...'
+    : flowName
+}
+
+export function createBodyErrorMessage(flowName: string): string {
+  const currDateTime = DateTime.now().toFormat('MMM dd yyyy, HH:mm:ss')
+  const bodyMessage = `
+    Dear user,
+    <br>
+    <br>
+    We have detected that your pipe: <strong>${flowName}</strong>, from plumber.gov.sg has failed as of ${currDateTime}.
+    <br>
+    Several scenarios could have caused this issue:
+    <ol>
+      <li>One of the apps you are using may be down.</li>
+      <li>Some parts of the pipe you have set up may not be working as intended.</li>
+    </ol>
+    <br>
+    Please check https://status.plumber.gov.sg/ to see if the status for Plumber is green or not. If one of the apps you are using are down, the plumber team would've already been alerted as well. 
+    <br>
+    <br>
+    If the pipe issue is urgent, please log in to plumber.gov.sg and check the pipe. Please retry the pipe under the executions tab subsequently. Otherwise, you may respond to this email and request assistance from the Plumber support team.
+    <br>
+    <br>
+    Regards,
+    <br>
+    Plumber Team
+  `
+  return bodyMessage
+}
+
+export async function sendErrorEmail(
+  flowName: string,
+  userEmail: string,
+  isTestRun: boolean,
+) {
+  const truncatedFlowName = truncateFlowName(flowName)
+  if (!isTestRun) {
+    sendEmail({
+      subject: `${truncatedFlowName} has execution errors`,
+      body: createBodyErrorMessage(truncatedFlowName),
+      recipient: userEmail,
```

Comment:
add `replyTo`

Reply:
(can we also test this works.... i dunno if zendesk will pick it up automatically)

Reply:
Yep will do so